packages/lab/src/tabs-next/TabNext.css

```
@@ -28,7 +29,7 @@
   min-width: var(--saltTabs-tab-minWidth, var(--salt-spacing-400));
   outline: none;
   position: var(--saltTabs-tab-position, var(--tabs-tab-position));
-  border-bottom-width: var(--tabs-activationIndicator-thumb-height);
+  border-bottom-width: var(--tabs-tab-border-width);
```

```suggestion
  border-bottom-width: var(--saltTabs-tab-border-width, var(--salt-size-bar));
```

local token unnecessary